test/mason/mason-argparse/MasonArgParseExample.chpl

```diff
@@ -7,7 +7,7 @@ module M {
         writeln(myConfigVar);
         
         writeln("Arguments Received from CL:");
-        for i in argsD {
+        for i in 1..argsD.high {
```

Comment:
I'd leave a comment saying you're skipping the 0th argument because it's the executable name and that can vary for multilocale runs